kc-cloudtrust-testsuite/kc-cloudtrust-test-tools/src/main/java/io/cloudtrust/keycloak/test/AbstractInKeycloakTest.java

```diff
@@ -247,7 +247,7 @@ public Map<String, List<String>> getUserAttributes(String realmName, String user
     }
 
     public List<String> getUserAttribute(String username, String attributeName) {
-        return getUserAttribute(username, attributeName);
+        return getUserAttribute(defaultRealmName, username, attributeName);
```

Comment:
It is a nice one :)

Reply:
Let's say that the callstack I had was clear with this...